src/services/etherpad/hooks/Pad.js

```diff
@@ -58,12 +59,17 @@ const before = {
 	find: [disallow()],
 	get: [disallow()],
 	create: [
-		globalHooks.hasPermission('TOOL_CREATE'),
-		injectCourseId,
-		globalHooks.restrictToUsersOwnCourses,
-		getGroupData,
-		restrictOldPadsToCourse,
-	],
+		iffElse(
+		  globalHooks.hasPermission(['TOOL_CREATE', 'TOOL_CREATE_ETHERPAD']),
+		  [
+			injectCourseId,
+			globalHooks.restrictToUsersOwnCourses,
+			getGroupData,
+			restrictOldPadsToCourse,
+		  ],
+		  []
+		),
```

Comment:
because hasPermission supports array, it will be true if any of the permissions is true
so don't need iffElse, but just to list both permissions
/**
 * @param  {string, array[string]} inputPermissions
 * @returns resolves if the current user has ANY of the given permissions
 */
const hasPermission = (inputPermissions) => {
	const permissionNames = typeof inputPermissions === 'string' ? [inputPermissions] : inputPermissions;

	return (context) => {
...

```suggestion
		globalHooks.hasPermission(['TOOL_CREATE', 'TOOL_CREATE_ETHERPAD']),
		injectCourseId,
		globalHooks.restrictToUsersOwnCourses,
		getGroupData,
		restrictOldPadsToCourse,
```

Reply:
Done